nagios-service/check_wpscan.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Hostname argument
+hostname="$1"
+
+# API key argument
+api_key="$2"
+
+# Execute WPScan and capture the output
+wpscan_output=$(wpscan --url $1 --api-token $api_key --format json)
+
+return_code="$?"
+
+# Check if WPScan command executed successfully
+if [ $return_code -eq 1 ]; then
+  echo "WPScan command failed with the following output:"
+  echo $wpscan_output
+  exit 1
+fi
+
+if [ "$return_code" -gt 1 ]; then
+  echo "CRITICAL: $return_code vulnerabilities found."
+  exit 2
+else
```

Comment:
are we certain about the return code being the number of vulns? I've tested this against a fresh JN site and I'm getting `[07-07-2023 12:59:01] SERVICE ALERT: <my-hostname>;WPScan;CRITICAL;SOFT;2;CRITICAL: 3 vulnerabilities found.` -- but when I run the same command manually it detects zero vulns? 🤔

Reply:
I remembered reading an issue that they were considering returning the number of vulns as the exit code, I just looked at the code and they just return an indicator if it's vulnerable or not, I'll do some `jq` processing to see the output instead.